tcms/testruns/templates/testruns/get.html

```diff
@@ -156,6 +156,7 @@ <h2 class="card-pf-title">
                                         <button type="button" class="btn btn-default dropdown-toggle" id="filter-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">{% trans "Change status" %} <span class="caret"></span></button>
                                         <ul class="dropdown-menu">
                                             {% for status in execution_statuses %}
+                                                {% comment %} TODO: attach a handler than returns false, so that it does no go to the top of the page {% endcomment %}
```

Comment:
Drop the last commit, b/c this is already in the backlog. We don't need extra commits with unfinished items as TODO comments.

And it will be ready to merge.

Reply:
drop the commit along with the changes, or just drop the commit but leave the comment?

Reply:
when you drop the commit all of the changes introduced with it will go away. So I mean remove the commit and this comment b/c it's useless.

Reply:
this was then state of the PR 3 weeks ago. it could have been merged then without the comment and the commit

Reply:
It couldn't have been merged b/c we've got pending question: https://github.com/kiwitcms/Kiwi/pull/1559#discussion_r456820234  but now I see that I mistyped option 3). It should have been "add more TODOs in the backlog" only, not in the code.